interface/src/framework/mqtt/MqttSettingsForm.tsx

```diff
@@ -219,15 +219,15 @@ const MqttSettingsForm: FC = () => {
               />
             </Grid>
             {data.ha_enabled && (
-              <Grid item xs={6}>
+              <Grid item xs={2}>
                 <ValidatedTextField
                   name="discovery_prefix"
                   label={LL.MQTT_PUBLISH_TEXT_4()}
                   fullWidth
                   variant="outlined"
                   value={data.discovery_prefix}
                   onChange={updateFormValue}
-                  margin="normal"
+                  size="small"
```

Comment:
making it small is not consistent with the other sizes, it also renders weird and the text is cutoff in English

Reply:
IMHO, a `size="small"` style rectangle looks better than a normal (large) one when is displayed in the line with the option name. Don't you think? Also, the same style was already used in the entity language option before.